pyfar/classes/audio.py

```diff
@@ -1249,3 +1249,92 @@ def _divide(a, b):
 
 def _power(a, b):
     return a**b
+
+
+def match_fft_norm(fft_norm_1, fft_norm_2, division=False):
+    """
+    Helper function to determine the fft_norm resulting from an
+    arithmetic operation of two signals.
+
+    For addition, subtraction and multiplication:
+    Either: one signal has fft_norm ``'none'`` , the results gets the other
+    norm.
+    Or: both have the same fft_norm. Other combinations raise an error.
+
+    For division:
+    Either: the denominator (fft_norm_2) is ``'none'``, the result gets the
+    fft_norm of the numerator (fft_norm_1).
+    Or:
+    ###  more discussion needed  ###
+
+    Parameters
+    ----------
+    fft_norm_1 : str, ``'none'``, ``'unitary'``, ``'amplitude'``, ``'rms'``,
+    ``'power'`` or ``'psd'``
+        First fft_norm for matching.
+    fft_norm_2 : str, ``'none'``, ``'unitary'``, ``'amplitude'``, ``'rms'``,
+    ``'power'`` or ``'psd'``
+        Second fft_norm for matching.
+    division : bool
+        ``False`` if arithmetic operation is addition, subtraction or
+        multiplication;
+        ``True`` if arithmetic operation is division.
+
+    Returns
+    -------
+    fft_norm_result : str, ``'none'``, ``'unitary'``, ``'amplitude'``,
+    ``'rms'``, ``'power'`` or ``'psd'``
+        The fft_norm resulting from arithmetic operation.
+    """
+
+    # check if fft_norms are type string
+    if isinstance(fft_norm_1, str) is False:
+        raise TypeError("Parameter fft_norm_1 must be type str.")
+    if isinstance(fft_norm_2, str) is False:
+        raise TypeError("Parameter fft_norm_2 must be type str.")
```

Comment:
You can omit this, because of the next check. If it is not a string the next will already raise a ValueError which would be fine for me.

Reply:
check, I wanted the raised errors to be as explicit as possible, but are absolutely fine with your suggestion.
I will implement it in my next commit.